reo/src/load_profile.py

```diff
@@ -777,3 +817,30 @@ def _account_for_existing_pv(self, pvs, years):
             native_load = [i + j for i, j in zip(self.unmodified_load_list, existing_pv_kw_list)]
             return native_load, existing_pv_kw_list
         return copy.copy(self.unmodified_load_list), existing_pv_kw_list
+
+def get_climate_zone(latitude, longitude):
```

Comment:
It looks we could use this function in `BuiltInProfile.city` to cut down on redundant code: maybe we should move `get_climate_zone` to utilities.py? I can do this if you agree.

Reply:
I did this.